userspace/libpman/src/sc_set.c

```diff
@@ -69,7 +93,54 @@ int pman_enforce_sc_set(bool *sc_set) {
 		sched_prog_exec = true;
 	}
 
-	/* Enable desired tracepoints */
+	if(sys_exit) {
+		sys_enter_connect = sc_set[PPM_SC_CONNECT];
+		if(sys_enter_connect) {
+			sys_enter_socketcall = true;
```

Comment:
`socketcall` TOCTOU mitigation program is only attach to support TOCTOU mitigation for `connect`.